### ISSUE TITLE
Invalid Version on kong enterprise edition

### ISSUE THREAD

**Issue by kjsteuer** [mybuilder/kongfig#109]
We recently started getting the following error on our konfig apply
```
...
TypeError: Invalid Version: 0.29-enterprise-edition
14:30:38
TypeError: Invalid Version: 0.29-enterprise-edition
14:30:38
at new SemVer (/usr/local/lib/node_modules/kongfig/node_modules/semver/semver.js:293:11)
14:30:38
at compare (/usr/local/lib/node_modules/kongfig/node_modules/semver/semver.js:566:10)
14:30:38
at Function.gte (/usr/local/lib/node_modules/kongfig/node_modules/semver/semver.js:615:10)
14:30:38
at _migrateApiDefinitionToVersion (/usr/local/lib/node_modules/kongfig/lib/migrate.js:55:34)
14:30:38
at /usr/local/lib/node_modules/kongfig/lib/migrate.js:67:19
14:30:38
at _callee3$ (/usr/local/lib/node_modules/kongfig/lib/core.js:193:63)
14:30:38
at tryCatch (/usr/local/lib/node_modules/kongfig/node_modules/regenerator-runtime/runtime.js:65:40)
14:30:38
at Generator.invoke [as _invoke] (/usr/local/lib/node_modules/kongfig/node_modules/regenerator-runtime/runtime.js:303:22)
14:30:38
at Generator.prototype.(anonymous function) [as next] (/usr/local/lib/node_modules/kongfig/node_modules/regenerator-runtime/runtime.js:117:21)
14:30:38
at step (/usr/local/lib/node_modules/kongfig/lib/core.js:45:273)
```

One thing that is interesting is kongfig apply works fine in some of our environments running 0.29-enterprise-edition.  0.29 is the latest enterprise version, it maps to 0.11 CE.

We are running npm version 1.3.0. I tried updating to npm 1.4.1  and the issue happens immediately vs at the end of the api diff checks.

Happy to make the required changes, I also found https://github.com/mybuilder/kongfig/pull/89 which seems related. Would that be a desirable approach?

**Comment by CyExy** [comment]
Probably the easiest fix is to strip postfix before comparing the version because it is irrelevant for the purpose we are using it for.

**Comment by CyExy** [comment]
Fixed in `kongfig@1.4.2`
